### DISCUSSION TITLE
Cannot decode byte '\xef': Data.Text.Internal.Encoding.decodeUtf8: Invalid UTF-8 stream

### DISCUSSION THREAD

**Discussion by nphard** [PostgREST/postgrest#1946] [Q&A] [answered]
* PostgreSQL version: (if using docker, specify the image)  12.3 docker image
* PostgREST version: (if using docker, specify the image) 7.0.1 image
* Operating system:  ubuntu linux 20.02

### Description of issue

(Expected behavior vs actual behavior)
We have this problem when a specific query happens: 
the url:  `fc_users?or=(name.ilike.%ef%,email.ilike.%ef%)&role=in.(student,teacher,gk_admin,super_admin,support)&order=last_modified.desc&last_modified=gte.2000-01-01&last_modified=lte.2100-01-01`

and the postgrest will emit errors : 
```
Cannot decode byte '\xef': Data.Text.Internal.Encoding.decodeUtf8: Invalid UTF-8 stream 
```

but from the postgres we can't find any logs.  

if there any configuration that could fix this error, so we could display the actual problem?  it seems like some decoding problem for the messages.

**Comment by steve-chavez** [comment]
We had a similar issue on https://github.com/PostgREST/postgrest/issues/1585. That one was fixed on [v8.0.0](https://github.com/PostgREST/postgrest/releases/tag/v8.0.0), could you try upgrading and see if that fixes the problem?

**Comment by wolfgangwalther** [comment]
This might be related to `%ef` in the query string, too? I assume the `%` needs to be encoded as `%25`, so the query URL should be:

```
 fc_users?or=(name.ilike.%25ef%25,email.ilike.%25ef%25)&role=in.(student,teacher,gk_admin,super_admin,support)&order=last_modified.desc&last_modified=gte.2000-01-01&last_modified=lte.2100-01-01
```

**Comment by nphard** [comment]
to upgrade to v8.0.0 ,  it seems there are some broken changes for the api backend,  it might require some extra work for the upgrade.  we will double check with that.

**Comment by wolfgangwalther** [comment, marked as answer]
One more note regarding your request URI / query: When using the `like.` / `ilike.` operators, you have to replace `%` with `*`, see [the docs](https://postgrest.org/en/v7.0.0/api.html#operators). Pretty sure you don't have to upgrade to fix this.
